docker/README.md

```diff
@@ -123,3 +139,34 @@ server {
 }
 ```
 
+## Installing lizmap modules 
+
+### Install from archive
+
+- Mount `/www/lizmap/lizmap-modules/` at a location on the host or a from a named volume.
+- Get the archive of the lizmap module and extract it in the `/www/lizmap/lizmap-module/` or 
+  Add the module configuration snippet in the appropriate snippets directory (see the module documentation for  the configuration files to modify)
+- Add `<module_name>.access =2` to the `[modules]` section of `localconfig.ini.php` (which should be locatad in your `<install_dir>/lizmap/var/lizmap-config` directory
+- Restart the container or run `php /www/lizmap/install/installer.php` from inside the container
+
+### Install from composer packages
+
+- Mount `/www/lizmap/my-packages/` at a location on the host or a from a named volume.
+- From inside the container, install your module with the command `lizmap-install-modules <package-name>`
+- Add the module configuration snippet in the appropriate snippets directory (see the module documentation for  the configuration files to modify)
+- Restart the container or run `php /www/lizmap/install/installer.php` from inside the container
+
+### Install for module development
+
+- Mount `/www/lizmap/lizmap-modules/` at the location of your module sources.
+  Add the module configuration snippet in the appropriate snippets directory (see the module documentation for  the configuration files to modify)
+- Add `<module_name>.access =2` to the `[modules]` section of `localconfig.ini.php` (which should be locatad in your `<install_dir>/lizmap/var/lizmap-config` directory
```

Comment:
same comment as above. Setting the access value depends of the module.

Reply:
fixed